### PR TITLE
Add SensitiveParameter attribute

### PR DESCRIPTION
**Pull request by ev-gor**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #89

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
```
Comment (vjik) [inline composer.json]: Not need to bump PHP version.

**Review by vjik — CHANGES_REQUESTED** [review]
add line to changelog

File: composer.json (hunk 1 above)
Comment (ev-gor) [reply composer.json]: Sorry, but why we don't need to update PHP version, if SensitiveParameter was added only in v. 8.2?

Comment (samdark) [reply composer.json]: I would be safely ignored in other versions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## 3.1.2 under development
 
-- no changes in this release.
+- Enh: Add SensitiveParameter attribute (@ev-gor)
```
Comment (vjik) [inline CHANGELOG.md]:
```suggestion
- Enh #90: Use `SensitiveParameter` attribute to mark sensitive parameters (@ev-gor)
```

File: src/IdentityWithTokenRepositoryInterface.php [hunk 3]
```diff
@@ -19,5 +19,5 @@ interface IdentityWithTokenRepositoryInterface
      * @return IdentityInterface|null The identity object that matches the given token. Null should be returned if such
      * an identity cannot be found or the identity is not in an active state (disabled, deleted, etc.)
      */
-    public function findIdentityByToken(string $token, string $type = null): ?IdentityInterface;
+    public function findIdentityByToken(#[\SensitiveParameter] string $token, string $type = null): ?IdentityInterface;
```
Comment (vjik) [inline src/IdentityWithTokenRepositoryInterface.php]:
```suggestion
    public function findIdentityByToken(string $token, string $type = null): ?IdentityInterface;
```

Attribute not need in interface. Attribute should be added in implementation. Example: https://3v4l.org/KNALl

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
@ev-gor Why you close PR?
It looks good.

**Pull request closed without merging**